README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## This is an autobiography from Yinsu Han for HW01.
+My name is **Yinsu Han**. I am a rising sophomore in the college, planning to major in **mathematics**. I am always excited to learn new things. Here are several things I enjoy apart from learning:
```

Comment:
Nice use of bold font! I recommend including a blank line between Markdown elements such as headers and plain text. It avoids rendering issues in more complex documents